evergreen/config.yml

```diff
@@ -1693,72 +1582,6 @@ buildvariants:
   - name: long-running-tests
   - name: swift-build-and-test
 
-- name: macos-1100-x64-asan
-  display_name: "MacOS 11.0 x86_64 (ASAN)"
-  run_on: macos-1100
-  expansions:
-    cmake_url: "https://s3.amazonaws.com/static.realm.io/evergreen-assets/cmake-3.26.3-macos-universal.tar.gz"
-    cmake_bindir: "./cmake_binaries/CMake.app/Contents/bin"
-    cmake_toolchain_file: "./tools/cmake/xcode.toolchain.cmake"
-    cmake_generator: Xcode
-    max_jobs: $(sysctl -n hw.logicalcpu)
-    xcode_developer_dir: /Applications/Xcode13.1.app/Contents/Developer
-    extra_flags: -DCMAKE_SYSTEM_NAME=Darwin -DCMAKE_OSX_ARCHITECTURES=x86_64
-    cmake_build_type: RelWithDebInfo
-    enable_asan: On
-  tasks:
-  - name: compile_test
-
-- name: macos-1100-x64-tsan
-  display_name: "MacOS 11.0 x86_64 (TSAN)"
-  run_on: macos-1100
-  expansions:
-    cmake_url: "https://s3.amazonaws.com/static.realm.io/evergreen-assets/cmake-3.26.3-macos-universal.tar.gz"
-    cmake_bindir: "./cmake_binaries/CMake.app/Contents/bin"
-    cmake_toolchain_file: "./tools/cmake/xcode.toolchain.cmake"
-    cmake_generator: Xcode
-    max_jobs: $(sysctl -n hw.logicalcpu)
-    xcode_developer_dir: /Applications/Xcode13.1.app/Contents/Developer
-    extra_flags: -DCMAKE_SYSTEM_NAME=Darwin -DCMAKE_OSX_ARCHITECTURES=x86_64
-    cmake_build_type: RelWithDebInfo
-    enable_tsan: On
-  tasks:
-  - name: compile_test
-
-- name: macos-1100-arm64-asan
-  display_name: "MacOS 11 arm64 (ASAN)"
-  run_on: macos-1100-arm64
-  expansions:
-    cmake_url: "https://s3.amazonaws.com/static.realm.io/evergreen-assets/cmake-3.26.3-macos-universal.tar.gz"
-    cmake_bindir: "./cmake_binaries/CMake.app/Contents/bin"
-    cmake_toolchain_file: "./tools/cmake/xcode.toolchain.cmake"
-    cmake_generator: Xcode
-    max_jobs: $(sysctl -n hw.logicalcpu)
-    xcode_developer_dir: /Applications/Xcode13.1.app/Contents/Developer
-    extra_flags: -DCMAKE_SYSTEM_NAME=Darwin -DCMAKE_OSX_ARCHITECTURES=arm64
-    cmake_build_type: RelWithDebInfo
-    enable_asan: On
-  tasks:
-  - name: compile_test
-
-- name: macos-1100-arm64-tsan
```

Comment:
this is the only builder which was problematic from what i know: it's a combination of tsan issue and macos11 arm64 clang. Is there anything else i'm not aware of?

Also, i had https://github.com/realm/realm-core/pull/7472 to make it work most likely. Do you have some other plans for this issue?

Reply:
I've put back the other builders for now since I think you're right that this is the only buggy one.

Reply:
Re #7472, I don't think we should switch to using ninja and I'm about to get xcode 15 rolled out so switching to macos 13 doesn't seem worth it.

Reply:
#7472 was just a temporal fix for now specifically for sanitizers on macos. We have significant number of ThreadSanitizer:DEADLYSIGNAL failures in the last two months on the ci on almost every build. The macos 13 clang is also able to catch #7583, which slipped through into master recently. I'm just afraid that migration will take even more time... It was really painful to even introduce this tsan builder in the first place due to many accumulated issues.